### PR TITLE
network-filesystems: Add section.

### PR DESCRIPTION
**Pull request by flexibeast**
This PR is based on, and supersedes, #382.

### CODE REVIEW

File: src/config/network-filesystems.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+Mounting options can be found in
+[mount.nfs(8)](https://man.voidlinux.org/mount.nfs.8), while unmounting options
+can be found in [umount.nfs(8)](https://man.voidlinux.org/umount.nfs.8).
+
+For example, to connect `/volume` on a server at `192.168.1.99` to a directory
+`/mnt/volume` on your local system:
+
+```
+# mkdir /mnt/volume
```
Comment (ericonr) [inline src/config/network-filesystems.md:31]: I don't know if this `mkdir` is necessary, but I'm not fully sure. It's general information about mounting stuff that the directory needs to exist so you can mount stuff to it.

File: src/config/network-filesystems.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+To have the directory mounted when the system boots, add an entry to
+[fstab(5)](https://man.voidlinux.org/fstab.5):
+
+```
+192.168.1.99:/volume /mnt/volume nfs rw,hard,intr 0 0
+```
+
+Refer to [nfs(5)](https://man.voidlinux.org/nfs.5) for more information about
+the available options.
```
Comment (ericonr) [inline src/config/network-filesystems.md:43]: "for information about the avaiable mounting options" ?

File: src/config/network-filesystems.md (hunk 1 above)
Comment (flexibeast) [reply src/config/network-filesystems.md]: Fair point; i've just updated the PR. Ok?

File: src/config/network-filesystems.md (hunk 2 above)
Comment (flexibeast) [reply src/config/network-filesystems.md]: Done.

**Comment by flexibeast** [comment]
@ericonr: Ok to merge?

**Comment by ericonr** [comment]
LGTM! Thanks!

**Pull request merged**